sparsesim/src/lib.rs

```diff
@@ -231,12 +260,17 @@ impl QuantumSim {
         self.check_joint_probability(&locs)
     }
 
+    /// Checks the internal state of the given qubit and returns true only if the given qubit is in exactly the |0⟩ state.
+    pub fn qubit_is_zero(&mut self, id: usize) -> bool {
+        self.joint_probability(&[id]).is_nearly_zero()
```

Comment:
I guess handled here then.

Reply:
Yes, that's right. The `NearlyZero` trait is just an internal helper to avoid bad results of exact floating point equality. So the implementation moved but did not change.